### ISSUE TITLE
Could a timeout option be provided for es ?

### ISSUE THREAD

**Issue by iamoyjj** [matthewfranglen/postgres-elasticsearch-fdw#4]
```
        self.client = Elasticsearch(
            [
                {
                    "host": options.get("host", "localhost"),
                    "port": int(options.get("port", "9200")),
                }
            ],timeout=int(options.get("timeout", "10"))
        )
```

**Comment by matthewfranglen** [comment]
Thanks for opening this ticket. I'll look into it tonight.

**Comment by matthewfranglen** [comment]
I have added this option. You can use it like this:

```sql
CREATE FOREIGN TABLE articles_es
    (
        id BIGINT,
        title TEXT,
        body TEXT,
        query TEXT,
        score NUMERIC
    )
SERVER multicorn_es
OPTIONS
    (
        host 'elasticsearch',
        port '9200',
        index 'article-index',
        type 'article',
        rowid_column 'id',
        query_column 'query',
        score_column 'score',
        timeout '20'
    )
;
```

It is optional and defaults to 10 seconds. If you have any problems with it please raise another ticket.

**Comment by matthewfranglen** [comment]
I've released version 0.6.0 which includes this change.
